lib/pf/services/manager/haproxy.pm

```diff
@@ -58,6 +70,17 @@ sub generateConfig {
     $tags{'http'} = '';
     $tags{'mysql_backend'} = '';
     $tags{'var_dir'} = $var_dir;
+    my $bind_process = '';
+    if ($self->_number_cpus > 1) {
```

Comment:
My question was more, if there is only 1 cpu, the CPU tag will not be defined...
Will the template be generated properly ?